google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -1510,10 +1523,9 @@ def compose sources, destination, acl: nil, encryption_key: nil
         #   bucket = storage.bucket "my-todo-app"
         #   list_files_url = bucket.signed_url version: :v4
         #
-        def signed_url path = nil, method: nil, expires: nil, content_type: nil,
-                       content_md5: nil, headers: nil, issuer: nil,
-                       client_email: nil, signing_key: nil, private_key: nil,
-                       query: nil, version: nil
+        def signed_url path = nil, method: "GET", expires: nil, content_type: nil, content_md5: nil, headers: nil,
+                       issuer: nil, client_email: nil, signing_key: nil, private_key: nil, query: nil, scheme: "HTTPS",
+                       virtual_hosted_style: nil, bucket_bound_hostname: nil, version: nil
```

Comment:
Readability nit: one keyword argument per line.

Also in the `signer.signed_url` calls below, and in definitions of `signed_url` in subsequent files below.

Reply:
Aligned in Bucket, Project, File, and SignerV4.